### ISSUE TITLE
Docker Hub image is no longer an Automated Build

### ISSUE THREAD

**Issue by pataquets** [staaldraad/tcpprox#5]
While working with #4, I realized it.
```
$ docker search tcpprox
NAME                       DESCRIPTION                                     STARS               OFFICIAL            AUTOMATED
pataquets/tcpprox-src      tcpprox source build                            0                                       [OK]
[...]
linimbus/tcpproxy          Tcp protocol proxy tool                         0                                       [OK]
staaldraad/tcpprox         Simple TCP proxy. Allows you to intercept/in…   0                                       
vogsphar/tcpproxy          tcp proxy based on node.js                      0                                       [OK]
                                                 0                                       
```
Notice that there is no OK on the AB column.
DH have lost their triggering in the past, maybe you just need to trigger it manually to relink it, or tinker with the settings. In the worst case, you might need to delete the DH image and recreate it.

**Comment by staaldraad** [comment]
I'll have a look at it this weekend 👍

**Comment by staaldraad** [comment]
👋  so it is on autobuild, but the build fails for auth reasons. Fixing that now

![Screenshot 2019-12-07 at 12 37 04](https://user-images.githubusercontent.com/4200832/70374007-71dfd900-18ee-11ea-9e97-b2f1aaca3c58.png)

**Comment by staaldraad** [comment]
Re-authed and rebuilt. Autobuilds are on but it doesn't show up as `Automated` when you do `docker search` 🤷‍♂

**Comment by pataquets** [comment]
Thank you for taking a look at it, @staaldraad.
After reading your comment, I awaited just if there was some delay/propagation/whatever in the AB image push after building, but today I've checked `docker search` result again and, in fact, it doesn't displays the `[OK]` in the AB column. At first, it looks like a fault on Docker Hub's side.
However, after looking at it closely, I've found a possible cause: the image is actually not being built from the local source code in the Docker build context, but is `go get`'ed (i.e. fetched from a remote source, not the local files) . I'm not 100% sure of that being the problem, which would involve Docker Hub performing some deep checks on what source is being used for the build, but I cannot rule it out, though.
Moreover, fetching the source from remote, breaks the local development Docker workflow of code->build->run->repeat and it difficults the build source audit. I've even found the local source copy statement in the Dockerfile, but commented out. Maybe it slipped by? Just in case, a PR follows.
